components/Receipt.js

```diff
@@ -358,6 +362,12 @@ export class Receipt extends React.Component {
                     <div className="detail reference">
                       <label>Reference:</label> {this.getInvoiceReference()}
                     </div>
+                    {transactions.length === 1 && (
+                      <div>
+                        <label>Payment Method:</label>{' '}
+                        {`${transactions[0].paymentMethod.type} ${transactions[0].paymentMethod.name}`}
+                      </div>
+                    )}
```

Comment:
The payment method can be null in our data model. Out of safety:

```suggestion
                    {transactions.length === 1 && transactions[0].paymentMethod && (
                      <div>
                        <label>Payment Method:</label>{' '}
                        {`${transactions[0].paymentMethod.type} ${transactions[0].paymentMethod.name}`}
                      </div>
                    )}
```